assets/scss/base/form/_form.scss

```diff
@@ -227,6 +227,16 @@ Styleguide Form.date
   @include ie-lte(7) {
     zoom: 1;
   }
+
+  input[type="text"] {
```

Comment:
I'm still not convinced this is needed. Can you attach a before/after screenshot showing that it is?